backends/module.mk

```diff
@@ -493,7 +493,8 @@ MODULE_OBJS += \
 	imgui/imgui.o \
 	imgui/imgui_draw.o \
 	imgui/imgui_widgets.o \
-	imgui/imgui_tables.o
+	imgui/imgui_tables.o \
+	imgui/imgui_scummvm_fonts.o
```

Comment:
We prefer object files to be sorted alphabetically. However, this change does not belong to this PR...